io.openems.edge.common/src/io/openems/edge/common/test/AbstractComponentTest.java

```diff
@@ -76,6 +76,13 @@ public TestCase input(ChannelAddress address, Object value) {
 			this.inputs.add(new ChannelValue(address, value));
 			return this;
 		}
+		
+		public TestCase input(ArrayList<ChannelAddress> address, ArrayList<Object> value) {
+			for (int i = 0; i< address.size() ; i++) {
+				this.inputs.add(new ChannelValue(address.get(i), value.get(i)));
+			}		
+			return this;
+		}
```

Comment:
I don't think I like this change very much... I believe it makes the definition of input variables more chaotic. But we can discuss it. 

If you want to keep it, you should add a JavaDoc